packages/react/src/publication/infrastructure/PublicationCacheManager.ts

```diff
@@ -75,6 +75,7 @@ export class PublicationCacheManager implements IPublicationCacheManager {
           },
           ...this.variables,
         },
+        returnPartialData: true,
```

Comment:
looks like this is not needed anymore, @cesarenaldi your other PR (https://github.com/lens-protocol/lens-sdk/pull/857) solved the issue.
but shall we keep returnPartialData here? in case of cache manager maybe it's not a problem to return partial data? makes it less likely to return nothing